java/ql/test/query-tests/maven-dependencies/.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
Apologies, but shouldn't this file be removed?

Reply:
Wow, good catch. One file out of over a thousand!

Reply:
This was simply the furthest down file^^